contracts/.storage-layout

```diff
@@ -0,0 +1,92 @@
+=======================
```

Comment:
either add this file to gitigore or add precommit/ci pipeline to auto generate this